napari_assistant/_workflow_io_utility.py

```diff
@@ -20,6 +20,7 @@ def initialise_root_functions(workflow, viewer):
     """
     # find all workflow steps with functions which have root images as an input
     root_functions = wf_steps_with_root_as_input(workflow)
+    layer_names = [str(lay) for lay in viewer.layers]
```

Comment:
Hi Ryan @Cryaaa ,

just a quick question: After our discussion this week, do you want to change this to `layer.name` ?
```suggestion
    layer_names = [lay.name for lay in viewer.layers]
```

Reply:
Ahhh yes I think I might have changed it in a different branch by accident! I’ve checked that it works already though so you can merge it with the change you have suggested!